docs/policy/release/attestation.rego

```diff
@@ -0,0 +1,43 @@
+#
+# METADATA
+# title: Builtin attestation policies
+# description: >-
+#   This package is responsible for validating attestation signature and syntax.
```

Comment:
Perhaps something like this:

```suggestion
#   This package is responsible for validating attestation signature and syntax.
#   Note that all builtin policies are always included and cannot be skipped regardless of your policy configuration.
```

Reply:
I've added the note to the builtin collection and the two packages.